.github/dependabot.yml

```
@@ -1,40 +1,40 @@
-      time: "02:00"
-      timezone: "America/New_York"
-    open-pull-requests-limit: 10
-    labels:
-      - "dependencies"
-    groups:
-      minor-and-patch:
-        update-types:
-          - "minor"
-          - "patch"
-
+version: 2
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)